lib/commands/preload.ts

```diff
@@ -164,29 +165,27 @@ Can be repeated to add multiple certificates.\
 		const visuals = getVisuals();
 		const nodeCleanup = await import('node-cleanup');
 		const { instanceOf } = await import('../errors');
-
-		// Check image file exists
-		try {
-			const fs = await import('fs');
-			await fs.promises.access(params.image);
-		} catch (error) {
-			throw new ExpectedError(
-				`The provided image path does not exist: ${params.image}`,
-			);
-		}
+		let appDeviceType: string | undefined;
 
 		// balena-preload currently does not work with numerical app IDs
 		// Load app here, and use app slug from hereon
-		if (options.app && !options.app.includes('/')) {
```

Comment:
Was this intentionally blocking the org/app syntax?

Reply:
Yes. Before this PR, the purpose of this code block was to "convert" an app ID in the format `1301645`, or an app name in the format `test-rpi`, to an app slug in the format `gh_paulo_castro/test-rpi`, so as to resolve ambiguities using the logic in the CLI's implementation of `getApplication`: https://github.com/balena-io/balena-cli/blob/v12.44.21/lib/utils/sdk.ts#L30-L50
(Also, not sure if it's still the case, but at one point the `balena-preload` package would not accept numeric app IDs, and this code block would avoid that problem as well.)
The test `!options.app.includes('/')` would check whether the user had already specified a slug in the format `org/appname` on the command line, in which case this code block could be skipped.

In this PR, we are choosing to run this code block regardless of the format of `options.app`, as long as it is defined, in order to obtain `appDeviceType` in addition to addressing the concerns above, when they apply.